files/zh-cn/web/css/css_view_transitions/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: CSS 视图过渡
+slug: Web/CSS/CSS_view_transitions
+l10n:
+  sourceCommit: ee7620b98800392073f3c8f94a9b2d048191356d
+---
+
+{{CSSRef}}
+
+**CSS 视图过渡**（CSS view transitions）模块定义了[视图过渡 API](/zh-CN/docs/Web/API/view_Transition_API) 的行为，它允许开发人员在文档之内或在文档之间的不同状态之间创建动画过渡。此模块还定义了用于设置这些过渡样式的 CSS 属性和伪元素。
+
+## 参考
+
+### 属性
+
+- {{cssxref("view-transition-class")}}
+- {{cssxref("view-transition-name")}}
+
+### At 规则和描述符
+
+- {{cssxref("@view-transition")}}
+  - [`navigation`](/zh-CN/docs/Web/CSS/@view-transition#navigation) 描述符
+
+### 选择器和伪元素
+
+- {{cssxref(":active-view-transition")}}
+- {{cssxref(":active-view-transition-type()")}}
+- {{cssxref("::view-transition")}}
+- {{cssxref("::view-transition-image-pair()")}}
+- {{cssxref("::view-transition-group()")}}
+- {{cssxref("::view-transition-new()")}}
+- {{cssxref("::view-transition-old()")}}
+
+### 接口
+
+- {{domxref("CSSViewTransitionRule")}}
+- {{domxref("ViewTransition")}}
+  - {{domxref("ViewTransition.skipTransition()")}} 方法
+  - {{domxref("ViewTransition.updateCallbackDone")}}
+  - {{domxref("ViewTransition.ready")}}
+  - {{domxref("ViewTransition.finished")}}
+- {{domxref("Document.startViewTransition()")}} 方法
+
+## 指南
+
+- [使用视图过渡 API](/zh-CN/docs/Web/API/View_Transition_API/Using)
+
+  - : 解释如何创建视图过渡和自定义过渡动画，包括操纵活动的视图过渡。
+
+## 相关概念
+
+- {{domxref("PageRevealEvent", "pagereveal")}} 事件
+- {{domxref("PageSwapEvent", "pageswap")}} 事件
+- {{domxref("Document.visibilityState")}}
+
+- [CSS 动画](/zh-CN/docs/Web/CSS/CSS_animations)模块
+
+  - {{cssxref("animation")}}
+  - {{cssxref("@keyframes")}}
+  - {{domxref("CSSKeyframesRule")}}
+  - {{domxref("CSSStyleRule")}}
+  - [Web 动画 API](/zh-CN/docs/Web/API/Web_Animations_API)
+
+- [CSS transforms](/zh-CN/docs/Web/CSS/CSS_transforms) 模块
```

Comment:
https://www.w3.org/zh-hans/news/2013/filter-effects-and-css-transforms-drafts-published/

```suggestion
- [CSS 变换](/zh-CN/docs/Web/CSS/CSS_transforms)模块
```